### ISSUE TITLE
New development requirements

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gregko/fb2-to-epub-converter#9]
```
1) Provide more flexible error handling
```

Original issue reported on code.google.com by `alex.b...@gmail.com` on 11 Dec 2010 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
2) Create windows makefile. Both linux and windows makefiles should allow to 
set version of product in make command line.
3) Add font embeddability checking.
4) Add undocumented --autotest option. If this option is set, output file 
should depend only on input file and options and should not depend on date, 
random generator etc.
This option is necessary for future implementation of autotest procedures.

```

Original comment by `alex.b...@gmail.com` on 31 Dec 2010 at 5:12

**Comment by GoogleCodeExporter** [comment]
```
1,2,3,4 - Done

Fixed in version: 1.0_4
```

Original comment by `alex.b...@gmail.com` on 31 Dec 2010 at 10:39
- Changed state: **Done**
